ERCS/erc-7608.md

```diff
@@ -0,0 +1,63 @@
+---
+eip: 7608
+title: Count Contract
+description: Designed to manage and keep track of a numerical count.
+discussions-to: [to be created]
```

Comment:
You need to add a discussions topic to https://ethereum-magicians.org/ with a link to this PR.